extension/test/newman/setup.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs')
+const path = require('path')
+
+if (!process.argv[2] || !process.argv[3]) {
+  console.error('Usage: setupNewman your.email@mail.com your-password')
```

Comment:
you just want to copy paste this file from another extension, no need to modify at all. The reason for that is that we do not want to edit it every time we add an environment variable.

Reply:
copied from the ext-magento-user extension.

Reply:
I agree [this version](https://stash.localdev.cc/projects/SGX/repos/shopify-user/pull-requests/72/diff#extension/test/newman/setup.js) is more flexible, so we should use it for all extensions. Will also be added in https://github.com/shopgate/ext-magento-cart/pull/3